test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -2515,4 +2517,51 @@ class AcquisitionDetailsControllerSpec
     )
   }
 
+  def validateAcquisitionDetailsCheckYourAnswersPage(
+    acquisitionDetailsAnswers: CompleteAcquisitionDetailsAnswers,
+    doc: Document
+  ): Unit = {
+    val expectedAcquisitionMethodDisplayName = acquisitionDetailsAnswers.acquisitionMethod match {
+      case AcquisitionMethod.Bought       => messages("returns.acquisitionMethod.Bought")
+      case AcquisitionMethod.Inherited    => messages("returns.acquisitionMethod.Inherited")
+      case AcquisitionMethod.Gifted       => messages("returns.acquisitionMethod.Gifted")
+      case AcquisitionMethod.Other(value) => value
+    }
+
+    doc.select("#acquisitionMethod-answer").text() shouldBe expectedAcquisitionMethodDisplayName
+    doc.select("#acquisitionPrice-answer").text() shouldBe formatAmountOfMoneyWithPoundSign(
+      acquisitionDetailsAnswers.acquisitionPrice.inPounds()
+    )
+
+    if (acquisitionDetailsAnswers.improvementCosts === AmountInPence.zero) {
+      doc.select("#improvementCosts-answer").text shouldBe "No"
+    } else {
+      doc.select("#improvementCosts-answer").text shouldBe "Yes"
+      doc.select("#improvementCosts-value-answer").text shouldBe formatAmountOfMoneyWithPoundSign(
+        acquisitionDetailsAnswers.improvementCosts.inPounds()
+      )
+    }
+
+    if (acquisitionDetailsAnswers.acquisitionFees === AmountInPence.zero) {
+      doc.select("#acquisitionFees-answer").text shouldBe "No"
+    } else {
+      doc.select("#acquisitionFees-answer").text shouldBe "Yes"
+      doc.select("#acquisitionFees-value-answer").text shouldBe formatAmountOfMoneyWithPoundSign(
+        acquisitionDetailsAnswers.acquisitionFees.inPounds()
+      )
+    }
+
+    if (acquisitionDetailsAnswers.acquisitionDate.value.isBefore(LocalDate.of(1981, 1, 3))) {
+      doc.select("#rebasedAcquisitionPrice-answer").get(0).text() match {
+        case "Yes" =>
+          doc.select("#rebasedAcquisitionPrice-answer").get(1).text() shouldBe formatAmountOfMoneyWithPoundSign(
```

Comment:
you could do a similar thing with the other fields where you label the yes/no answer and the value answer differently to avoid doing `.get(0)` and `.get(1)`

Reply:
done, thx